command/monitor/monitor.go

```diff
@@ -0,0 +1,150 @@
+package monitor
```

Comment:
Instead of living in the command package can we move this to `helpers/monitor`?

Reply:
You bet